mkdocs_redirects/plugin.py

```diff
@@ -15,9 +13,9 @@
 def write_html(site_dir, old_path, new_path):
     """ Write an HTML file in the site_dir with a meta redirect to the new page """
     # Determine all relevant paths
-    old_path_abs = os.path.join(site_dir, old_path)
-    old_dir = os.path.dirname(old_path)
-    old_dir_abs = os.path.dirname(old_path_abs)
+    old_path_abs = os.path.join(site_dir, old_path).replace('\\', '/')
+    old_dir = os.path.dirname(old_path).replace('\\', '/')
```

Comment:
Files on POSIX could contain a backslash, don't need to break them.

```suggestion
    old_dir = pathlib.Path(old_path).parent.as_posix()
```

Reply:
I don't plan to introduce pathlib here.

Reply:
Then you're welcome to find a different way to not introduce this new problem.

Reply:
You are also welcome to suggest another way which also keeps Python 2.x compatibility.

Reply:
Why would you need Python 2.x compatibility?
[MkDocs runs on Python 3](https://www.mkdocs.org/user-guide/installation/)!